src/index.html

```diff
@@ -36,6 +36,7 @@ <h1>
         <li><a href="https://github.com/remotestorage">GitHub</a></li>
         <li><a href="https://kiwiirc.com/client/irc.freenode.net/#remotestorage">Chat</a></li>
         <li><a href="https://twitter.com/remotestorage_">Twitter</a></li>
+        <li><a href="https://kosmos.social/@remotestorage">Mastodon</a></li>
```

Comment:
I think we should probably use the more general term for the network, where all users can follow our account, as long as their server supports ActivityPub:

```suggestion
        <li><a href="https://kosmos.social/@remotestorage">Fediverse</a></li>
```

Reply:
That's a good point!